mythril/laser/ethereum/state.py

```diff
@@ -283,7 +284,7 @@ def append(self, element):
             )
         super(MachineStack, self).append(element)
 
-    def pop(self, index=-1):
+    def pop(self, index=-1) -> BitVecNumRef:
```

Comment:
I think the stack contains BitVec expressions. Also I think BitVecNum is inherited from BitVec.

Reply:
Just found the inheritance diagram for Z3 types: https://z3prover.github.io/api/html/classz3py_1_1_expr_ref__inherit__graph.png

So I'll replace the specific hints with `ExprRef` for symbolic types. That should also cover BoolRefs